### PR TITLE
Better warnings + Version in minimap tooltip

### PR DESCRIPTION
**Pull request by blenderer**

### CODE REVIEW

File: Hardcore.lua [hunk 1]
```diff
@@ -754,6 +749,35 @@ function Hardcore:RequestTimePlayed()
...
+	end
+end
+
+function Hardcore:DisplayPlaytimeWarning(level)
+	local messageprefix = "\124cffFF0000"
+
+	if level <= 20 then
+		Hardcore:Print(messageprefix.."Detected that the player's addon active time is much lower than played time. If you have just installed the addon, start a new character.")
+	else
+		Hardcore:Print(messageprefix.."Detected that the player's addon active time is much lower than played time. If you have just installed the addon: consider starting a new character. Continuing on means you risk your lv 60, HC Verified Status.")
```
Comment (aaronma37) [inline Hardcore.lua:776]: At this point, would we really invalidate someones run if the percentage is too large. I imagine some people just dc a lot

**Review by aaronma37 — APPROVED** [review]

**Comment by aaronma37** [comment]
I think you wrote some unit tests. Should we add those for pater utility?

**Comment by blenderer** [comment]
> 

What is pater utility?

File: Hardcore.lua [hunk 2]
```diff
@@ -754,6 +749,35 @@ function Hardcore:RequestTimePlayed()
...
+		Hardcore:Print(messageprefix.."If you have had Hardcore 0.5.0 or greater installed since level 1, contact a mod and a record the rest of your run.")
```
Comment (RobertB4) [inline Hardcore.lua:777]: Not an English native speaker but I think there is an "a" too much.

`contact a mod and a record the rest of your run` -> `contact a mod and record the rest of your run`

If I am wrong just ignore this.

**Review by RobertB4 — APPROVED** [review]

**Comment by aaronma37** [comment]
> @aaronma37 What is pater utility?

Oops i meant later utility, as in to to test at a later time

**Pull request merged**